test/performance/shmem_perf_suite/common.h

```diff
@@ -36,6 +36,8 @@
 #include <time.h>
 #include <stdint.h>
 
+#include <bits/local_lim.h>
```

Comment:
What's this for?  IIRC including headers directly from `bits/` is bad for portability.

Reply:
This is for `HOST_NAME_MAX`. Initially, I added `limits.h`, but changed to this one. Should I keep `limits.h` instead?

Reply:
I think you want something similar to the solution in #653 
```
#include <limits.h>
#include <sys/param.h>

#ifdef MAXHOSTNAMELEN
#define SHMEM_INTERNAL_MAX_HOSTNAME_LEN MAXHOSTNAMELEN
#else
#define SHMEM_INTERNAL_MAX_HOSTNAME_LEN HOST_NAME_MAX
#endif
```